### PR TITLE
gTile GNOME extension described for SLED

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Are there any relevant issues/feature requests?

fixes https://jira.suse.com/browse/SLE-25081

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 SP0
- SLE 12
  - [x] SLE 12 SP5
  - [x] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/x11.xml [hunk 1]
```diff
@@ -179,6 +179,226 @@ enabled=true</screen>
...
+            secondary windows.
+          </para>
+        </step>
+        <step>
+          <para>
+            To move the current window to within the pre-defined tiling of the
+            screen, use the arrow keys.
+          </para>
+          <para>
+            To shrink or grow the number of tiles, use
```
Comment (federicomenaquintero) [inline xml/x11.xml:359]: "the number of tiles that the window occupies", perhaps?  To clarify that it doesn't change the number of available tiles, just the window size.

File: xml/x11.xml [hunk 2]
```diff
@@ -179,6 +179,226 @@ enabled=true</screen>
...
+           xlink:href="https://extensions.gnome.org/extension/28/gtile/"/>.
+            If you see a warning about a missing browser extension, click the
+            link and follow your Web browser's instruction on how to install and
+            enable the &gnome; Shell extension.
+          </para>
+          <figure>
+            <title>Warning about missing &gnome; Shell extension</title>
+            <mediaobject>
+              <imageobject role="fo">
+                <imagedata fileref="gtile-browser-warning.png" width="75%"/>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/x11.xml:217]: welcome :-)

File: xml/x11.xml [hunk 3]
```diff
@@ -179,6 +179,226 @@ enabled=true</screen>
...
+              <imageobject role="html">
+                <imagedata fileref="gtile-tray-icon.png" width="75%"/>
+              </imageobject>
+              <textobject role="description"><phrase>gTile tray icon</phrase>
+              </textobject>
+            </mediaobject>
+          </figure>
+        </step>
+      </procedure>
+      <!-- i dont think we need this
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/x11.xml:268]: strange, it worked for me like a breeze...

File: xml/x11.xml (hunk 1 above)
Comment (tbazant) [inline xml/x11.xml:359]:
```suggestion
            To shrink or grow the number of tiles that the window occupies, use
```

**Review by tbazant — COMMENTED** [review]
suggested minor changes

File: xml/x11.xml [hunk 4]
```diff
@@ -179,6 +179,170 @@ enabled=true</screen>
...
+        </step>
+        <step>
+          <para>
+            Select gTile and click on <guimenu>Settings</guimenu>.
+          </para>
+        </step>
+        <step>
+          <para>
+            Define a preset and hotkey to easily switch between frequently used
+           window layouts.
```
Comment (dariavladykina) [inline xml/x11.xml:336]:
```suggestion
           Define a preset and a key combination to easily switch between
           frequently used window layouts.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

**Pull request merged**